website/docs/scaffolds/taco-shop.mdx

```diff
@@ -19,7 +19,7 @@ In a rush? You can follow the steps below to get up and running immediatley
 
 1. `taq scaffold https://github.com/ecadlabs/taqueria-scaffold-taco-shop taco-shop`
 2. `cd taco-shop`
-3. `npm run setup`
+3. `npm run start`
 
 ## Overview
```

Comment:
Although not necessary for merging this in, in the future we should probably add a heading and section for provisioning.